osquery/tables/system/windows/chassis_info.cpp

```diff
@@ -0,0 +1,69 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <osquery/logger.h>
+#include <osquery/sql.h>
+#include <osquery/system.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/windows/wmi.h"
+
+namespace osquery {
+namespace tables {
+
+QueryData genChassisInfo(QueryContext& context) {
+  QueryData results;
+
+  WmiRequest wmiSystemReq("select * from Win32_SystemEnclosure");
+  const auto& wmiResults = wmiSystemReq.results();
+  if (wmiSystemReq.getStatus().ok()) {
```

Comment:
Flip this and early return

Reply:
QQ about the formatting flipping - this is the same format I was asked to follow when i submitted the ntdomains table. Happy to flip the logic, but I'm wondering which one is better for consistency? I'll need to go back and re-arrange some of my other tables depending where we go with the standards.

Reply:
@muffins @fmanco 

We generally prefer outdenting as much as possible right?

Reply:
Pushed a change with a flip and early return. Feel free to take a look. I wanted to make sure I had that right before I addressed the other changes I'll make before it's ready:
-Rename boolean to something more reasonable (endrun?)